scripts/parser.js

```diff
@@ -11,7 +11,7 @@ const mbo = new MBO({
  * Iterates through provided class information from Mindbody API
  * and updates or inserts jobs to the database
  */
-async function processResponse(err,data){
+async function processClasses(err,data){
```

Comment:
Same thing with this call back function, we can keep the funciton this way for now but in the future we might do a refactor of this function so we can separate out the funcitonalities, but for now, let's change the name of the function to something like `_parseClasses` so it is easier to understand vs `processClasses`.